fmperf/utils/constants.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+REQUESTS_DIR = os.environ.get("REQUESTS_DIR", "")
```

Comment:
Should we maybe make the default:
```
REQUESTS_DIR = os.environ.get("REQUESTS_DIR", ".")
```
So explicitly the default is to use the currently directory? I guess empty string may have same effect, not 100% sure.

Reply:
I agree that this is the best option. Or would you prefer to keep `/requests/` as the default for backward compatibility?

Reply:
Hmm yeah, maybe that would be the least intrusive thing actually. That or we could just add it to the example env file here: https://github.com/fmperf-project/fmperf/blob/main/.env.example

Reply:
done